src/main/java/org/skriptlang/skript/lang/experiment/Experiment.java

```diff
@@ -56,6 +57,20 @@ default boolean isKnown() {
 	 */
 	SkriptPattern pattern();
 
+	/**
+	 * @return The display name for this experiment.
+	 */
+	default @NotNull String displayName() {
+		return "";
```

Comment:
```suggestion
		return codeName();
```
Might be nice to have a reasonable fallback for this one.